app/scripts/services/Timer.js

```diff
@@ -2,36 +2,60 @@
   function Timer($interval) {
     var Timer = {};
     var stop;
-    var counterLength = 1500;
+    var SESSION_LENGTH = 1500;
+    var BREAK_LENGTH = 300;
 
     Timer.buttonText = "Let's Get Started";
-    Timer.counter = counterLength;
+    Timer.counter = SESSION_LENGTH;
+    Timer.onBreak = false;
 
     Timer.startSession = function () {
-      this.isTimeRunning = true;
+      Timer.isTimeRunning = true;
       this.timeStarted = true;
```

Comment:
My only piece of feedback is that I was confused about the difference between `isTimeRunning` and `timeStarted` until I took a close look at the code. Might be worth thinking of other variable names to make it clear what each is doing, otherwise this might be a good place for a comment to improve readability! The comment would explain the difference between the two and what they're used for since they seem so similar on the surface.